src/core/packages/chrome/navigation/src/components/side_nav/panel.tsx

```diff
@@ -25,7 +28,7 @@ export interface SideNavPanelProps {
  *
  * TODO: pass ref to EuiPanel
  */
-export const SideNavPanel = ({ children }: SideNavPanelProps): JSX.Element => {
+export const SideNavPanel = ({ children, solutionId }: SideNavPanelProps): JSX.Element => {
```

Comment:
This doesn't seem great that we now need to pass the solutionId since the sidenav was almost Kibana agnostic before. I would consider replacing this with `feedbackSnippet:ReactNode` and would pass this from outside. 
I would even make feedback_snipper a separate package to keep the navigation package independent from Kibana. 
I'd, of course, leave the final design choice to @weronikaolejniczak.

Reply:
@Dosant I agree 👍🏻 I would:
- remove `solutionId` from the `<Navigation />` props,
- add a `sidePanelFooter` slot prop,
- move the feedback snippet to a separate package under `chrome` (or maybe there's a better spot for just one component),
- wherever the `<Navigation />` widget is rendered, pass the `<FeedbackSnippet />` through the `sidePanelFooter` slot (**consideration:** it has been mentioned by the solution teams that it'd be nice if the feedback snippet is reusable outside of the navigation context).

@angeles-mb If you need some more clarification on the approach, let me know and we can sync!

Reply:
> move the feedback snippet to a separate package under chrome (or maybe there's a better spot for just one component),

since the goal is to make feedback snippet reusable, I'd put it under shared/packages/shared-ux/feedback-snippet